Exiled.Events/EventArgs/SendingMessageEventArgs.cs

```diff
@@ -0,0 +1,52 @@
+// -----------------------------------------------------------------------
+// <copyright file="SendingMessageEventArgs.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.Events.EventArgs
+{
+    using System;
+
+    /// <summary>
+    /// Contains all the information after sending a C.A.S.S.I.E. message.
+    /// </summary>
+    public class SendingMessageEventArgs : EventArgs
```

Comment:
Wouldn't a closer name be better? E.g., if we decide to add an event for the admin chat (messages starting with `@` in RA)...

Reply:
Renamed it to SendingCassieMessage